app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -167,6 +168,7 @@ object AppRepository {
             executionContext: ExecutionContext = AppExecutionContext,
             databaseScope: DatabaseScope = DatabaseScope.of(executionContext, AppExceptionHandler(logging)),
             networkScope: NetworkScope = NetworkScope.of(executionContext, AppExceptionHandler(logging)),
+            okHttpClient: OkHttpClient = CustomHttpClient.createHttpClient(),
```

Comment:
`AppRepository#initialize` is invoked in `MyApp#onCreate` and in two test classes. `AppRepository#initialize` is overloaded because most of the parameters are only replaced in test scenarios.

Reply:
I see...possibly this is something for a separated issue, since testability and separation of concern could profit a lot from Dependency injection.

Reply:
Yes, let's chat about it at some point.